src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -124,6 +125,47 @@ public sealed partial class ConfidentialClientApplication
                 userAssertion);
         }
 
+        /// <inheritdoc />
```

Comment:
nit: I never understood `inheritdoc`. Do they actually work in .NET?

Reply:
Yes. Pulls documentation from base class or interface and merges it with any XML tags that are defined on the current member. We should use it.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc